src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -66,15 +64,6 @@ TypesMustExist : Type 'System.Reflection.MemberTypes' does not exist in the impl
 TypesMustExist : Type 'System.Reflection.ParameterModifier' does not exist in the implementation but it does exist in the contract.
```

Comment:
I'm looking at the issues locally. I understand what is happening with these reflection types. I will look over the rest and make sure I understand it and if there is work to be done I will file issues.